runtime/oti/j9protos.h

```diff
@@ -1133,6 +1133,52 @@ extern J9_CFUNC struct J9Method* jitResolveStaticMethodRef(J9VMThread *vmStruct,
 extern J9_CFUNC struct J9Method* jitResolveSpecialMethodRef(J9VMThread *vmStruct, J9ConstantPool *constantPool, UDATA cpOrSplitIndex, UDATA resolveFlags);
 extern J9_CFUNC struct J9Class * jitGetDeclaringClassOfROMField(J9VMThread *vmStruct, J9Class *clazz, J9ROMFieldShape *romField);
 
+typedef struct J9MethodFromSignatureWalkState {
+	char *className;
+	U_32 classNameLength;
+	struct J9JNINameAndSignature nameAndSig;
+	struct J9VMThread *vmThread;
+	struct J9ClassLoaderWalkState classLoaderWalkState;
+} J9MethodFromSignatureWalkState;
+
+/**
+ * Cleans up the iterator after matching all methods found.
+ *
+ * @param state				The inlialized iterator state
+ */
+extern J9_CFUNC void
+allMethodFromSignatureEndDo(J9MethodFromSignatureWalkState *state);
+
+/**
+ * Advances the iterator looking for the next method matching the full signature provided during initialization.
+ *
+ * @param state				The inlialized iterator state
+ *
+ * @return 					J9Method matching the full signature provided during initialization of the iterator in the
+ * 							class loader if it exists; NULL if no such method is found.
+ */
+extern J9_CFUNC struct J9Method*
+allMethodFromSignatureNextDo(J9MethodFromSignatureWalkState *state);
+
+/**
+ * Initializes the iterator state and returns the first method matching the supplied signature from any class loader.
+ *
+ * @param state				The iterator state which will be initialized.
+ * @param vm				The VM instance to look for methods in.
+ * @param flags				The flags forwarded to class loader iterator functions
+ * @param className			The class name including the package.
+ * @param classNameLength	The length (in number of bytes) of the class name
+ * @param methodName		The method name
+ * @param methodNameLength	The length (in number of bytes) of the method name
+ * @param methodSig			The method signature
+ * @param methodSigLength	The length (in number of bytes) of the method signature
+ *
+ * @return 					J9Method matching the full signature provided in the first class loader it is found; NULL if
+ * 							no such method is found.
```

Comment:
Please don't use tabs other than for leading indentation.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/70f787d25b95b995346cd4b3b411e35100b48f98..e6d4272c33428d33d17da82f1cddbbf549d82084).